Facepunch.Steamworks/SteamClient.cs

```diff
@@ -169,6 +170,11 @@ internal static void UnregisterCallback( IntPtr intPtr )
 		/// </summary>
 		public static AppId AppId { get; internal set; }
 
+		/// <summary>
+		/// Check if this SteamClient owns the specified lobby
+		/// </summary>
+		public static bool OwnsLobby( Lobby k ) => k.Owner.Id == SteamId;
```

Comment:
This and the one in friends is really the wrong way around. You'd add this function to the Lobby if anywhere.

Reply:
Ah, I do see that now. Thank you.